esercizio_carcere/index.js

```diff
@@ -0,0 +1,90 @@
+
+
+
+
+document.addEventListener('DOMContentLoaded', async ()=>{
+
+    const usersResponse = await fetch('http://dms.cyberdelia.eu/api/v1/user');
+    const usersRaw = await usersResponse.json();
+    // const users = usersRaw.filter((user) => {
+    //     return user.firstname && user.lastname;
+    // });
+
+    //destructuring 
+    // const users = usersRaw.filter(({ firstname, lastname}) => {
+    //     return firstname && lastname;
+    // });
+
+    //instant return arrow function
+    let usersInPrison = usersRaw.filter(({ firstname, lastname}) => (firstname && lastname) );
+    let freeUsers = [];
+    const prison = document.getElementById('prison');
+    const freedom = document.getElementById('freedom');
+    
+
+
+    const judgeAction = (action, user) => {
+        if(action == 'Scarcera'){
```

Comment:
In generale quando usi l'equality è una best pratics utilizzare lo strict equality. Dai un occhiata qui 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality